test/fixtures/build.json

```diff
@@ -7,5 +7,5 @@
   "version": "2.1",
   "fingerprints": ["a", "a.gz"],
   "artifacts": ["a/email.js"],
-  "recommended": []
+  "recommended": ["a/email.js"]
```

Comment:
how come these needed to be changed?

Reply:
if recommended was only passed an empty array, when we test it later in `assume(result.recommended )...`, result.recommended becomes undefined. This happens with artifacts as well, where an empty array past in results in undefined field in the result.